contrib/update-submodules.sh

```diff
@@ -1,11 +1,9 @@
 #!/bin/sh
-
 set -e
 
-WORKDIR=$(dirname "$0")
-WORKDIR=$(readlink -f "${WORKDIR}")
+GITDIR=$(git rev-parse --show-toplevel)
```

Comment:
Again, it will work for the PWD of the parent. We need both `WORKTREE`, and then calculate `GITDIR` accordingly to it

Reply:
Okay, I tried to fix it.